### PR TITLE
Additional extension helpers

### PR DESCRIPTION
**Pull request by Ralith**
Just some simple conveniences.

### CODE REVIEW

File: vulkano/src/instance/extensions.rs [hunk 1]
```diff
@@ -136,6 +141,14 @@ macro_rules! extensions {
             }
         }
 
+        impl<I> From<I> for $rawname
```
Comment (tomaka) [inline vulkano/src/instance/extensions.rs]: If you want to do that, you should implement the `FromIterator` trait instead.

Comment (Ralith) [inline vulkano/src/instance/extensions.rs]: Whoops, fixed!

File: vulkano/src/instance/extensions.rs [hunk 2]
```diff
@@ -136,6 +142,14 @@ macro_rules! extensions {
             }
         }
 
+        impl iter::FromIterator<CString> for $rawname {
```
Comment (tomaka) [inline vulkano/src/instance/extensions.rs]: Minor style: use `FromIterator` instead of `iter::FromIterator`.

**Comment by tomaka** [comment]
Will merge after nit.

**Comment by Ralith** [comment]
Tweaked as requested.

**Pull request merged**